src/main/java/edu/uz/dr/auctioneer/model/request/AuctionRequest.java

```diff
@@ -101,13 +98,17 @@ public void setUserDescription(final String userDescription) {
         this.userDescription = userDescription;
     }
 
+    public static String getDateFormat() {
```

Comment:
Jeśli pojo tester testuje settery i gettery to musi je mieć od wszystkich zmiennych a od zmiennej dateforrmat nie można było utworzyć i testy nie przechodziły